docs/ja/docs/advanced/index.md

```diff
@@ -0,0 +1,24 @@
+# ユーザーガイド 応用編
+
+## 追加機能
+
+[チュートリアル - ユーザーガイド](../tutorial/){.internal-link target=_blank}により、**FastAPI**の主要な機能は十分に理解できたことでしょう。
+
+以降のセクションでは、チュートリアルでは説明しきれなかったオプションや設定、および追加機能について説明します。
+
+!!! tip "豆知識"
+    以降のセクションは、 **必ずしも"応用編"ではありません**.
+
+    ユースケースによっては、中から解決策を見つけることができるでしょう。
```

Comment:
```suggestion
    様々なユースケースに応じたソリューションも説明します。
```
is this not better?

Reply:
uhhh,,, I feel it's too mean-spirited.

Reply:
I felt that "中から" made the sentence unnatural, so I modified it as follows, as long as it does not deviate too much from the original text. What do you think? https://github.com/tiangolo/fastapi/pull/5043/commits/cbff1fff189885a693280a38a71d6bb9bf63ad1a

Reply:
I understand the meaning, but that may be because I have read both the original and the translated text many times.
To improve this, we need the opinions of other reviewers.

Reply:
The current one looks no problem to me, but would be a little straightforward to point `them -> the next sections`.

```
< ユースケースによっては、解決策を見つけることができるでしょう。
> ユースケースによっては、その中から解決策を見つけられるかもしれません。
```